shell/common/shell.cc

```diff
@@ -1060,11 +1060,12 @@ void Shell::OnPlatformViewSetNextFrameCallback(const fml::closure& closure) {
   FML_DCHECK(task_runners_.GetPlatformTaskRunner()->RunsTasksOnCurrentThread());
 
   task_runners_.GetRasterTaskRunner()->PostTask(
-      [rasterizer = rasterizer_->GetWeakPtr(), closure = closure]() {
+      fml::MakeCopyable([rasterizer = rasterizer_->GetWeakPtr(),
+                         closure = std::move(closure)]() mutable {
```

Comment:
`closure` is a const reference. So `std::move(closure)` is still going to be doing a copy like before. We have had issues in the past where there were captures in the closure that could only be destroyed on specific threads were dropped when the closure was collected on an unexpected thread. The solution there is to use a capture by pointer and then collect it on the right thread. If this patch is doing anything, it is only delaying the collection in a non-deterministic manner.

Reply:
Thanks, I learned.